### DISCUSSION TITLE
option to move messages from issues to discusions

### DISCUSSION THREAD

**Discussion by ldijkman** [community/community#2863] [Discussions] [answered]
hello

verry good to add discussions

option to move messages from issues to discusions?

greet luberth
https://github.com/ldijkman

**Comment by becca** [comment, marked as answer]
You can convert any individual issue into a discussion - this functionality exists in the righthand sidebar of any issue. 

Additionally, you can also convert in bulk via labels: https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository#converting-issues-based-on-labels

**Reply by panva** [reply]
@becca there's something awry with bulk change via labels, while i have hundreds of issues with a given label, the UI shows me only one will be converted.

<img width="1242" alt="Screenshot 2020-12-09 at 11 42 31" src="https://user-images.githubusercontent.com/241506/101619964-3e112500-3a14-11eb-8bea-cd3b7d0a40e1.png">

<img width="616" alt="Screenshot 2020-12-09 at 11 42 53" src="https://user-images.githubusercontent.com/241506/101619976-40737f00-3a14-11eb-9a5d-377eea589410.png">

**Reply by becca** [reply]
is this because only one of them is open?

**Reply by panva** [reply]
1) you can see in the screenshot none are open
2) I would expect even closed issues to be transfered

**Reply by becca** [reply]
🤔 that is quite odd. I am not able to repro. @panva can you please share more information on the repository where this is happening and we can look into this?

**Reply by panva** [reply]
@becca https://github.com/panva/node-oidc-provider

**Reply by panva** [reply]
@becca and in [another repo](https://github.com/panva/jose) of mine I cannot get to bulk move action at all (now discussions are disabled but when i enable them, the bulk move action does not show up in the labels views)

<img width="1233" alt="Screenshot 2020-12-09 at 17 40 25" src="https://user-images.githubusercontent.com/241506/101658826-afb59700-3a45-11eb-81d9-8b472e58359e.png">

**Reply by becca** [reply]
@panva do you know your permission level for this repository?

**Reply by panva** [reply]
I am the owner of both.

**Reply by becca** [reply]
Only open issues can be converted and all of yours look to be closed. This is a gap in our documentation right now that I'll be sure to fix as soon as possible.

**Reply by panva** [reply]
@becca okay, but still - in the screenshot shown - i have 0 issues open, and yet the button shows up and says 1 issue will convert. So I guess there's still something awry lingering around.

**Reply by becca** [reply]
How were you able to have that module appear? I ask because it's only available after selecting `convert issues` and in your case you don't have that option since no issues are open.

**Reply by panva** [reply]
I enabled the discussions feature and opened the labels view, the button was then visible, even with no issues open.

**Reply by panva** [reply]
It is still the case:

<img width="513" alt="Screenshot 2020-12-14 at 17 22 57" src="https://user-images.githubusercontent.com/241506/102106616-14e7fe80-3e31-11eb-866b-353dc2e3ae26.png">
<img width="1246" alt="Screenshot 2020-12-14 at 17 22 53" src="https://user-images.githubusercontent.com/241506/102106627-17e2ef00-3e31-11eb-9f7c-10cf923f163e.png">

**Comment by maxandersen** [comment]
has this changed ? 

i'm owner of the jbangdev/jbang repo but I don't see any convert to discussion button - neither on individual issues nor in the label view.

**Reply by lesteve** [reply]
Same problem for me. For example on https://github.com/INRIA/scikit-learn-mooc/issues/153 I don't see the "Convert to discussion" option:
![image](https://user-images.githubusercontent.com/1680079/103520496-88e95580-4e77-11eb-9914-eac76e182146.png)

This problem was mentioned in https://github.com/github/feedback/discussions/2952#discussioncomment-259151 too.

**Reply by jpivarski** [reply]
I know that I used to have this feature, and I converted about 15 not-really-issues into Discussions: it was great!

Now it's missing, though. I expected to find the "Convert to Discussion" button under "Transfer Issue", but it's not there. Please bring it back! That was a very useful feature!

**Reply by arogozhnikov** [reply]
Same, just planned to transfer several issues into disccussions

**Reply by becca** [reply]
https://github.com/github/feedback/discussions/2924#discussioncomment-254592 The functionality exists but is currently hidden as we stabilize the feature to avoid any data being lost during conversions. We'll be notifying you all in the linked discussion when it's available again - the team is working on it right now.

**Comment by Sw333theart** [comment]
https://github.com/orgs/community/discussions/2863#discussioncomment-259545

**Comment by Sw333theart** [comment]
https://github.com/orgs/community/discussions/2863#discussioncomment-259545
